libs/wlgen/wlgen/rta.py

```diff
@@ -133,7 +133,8 @@ def calibrate(target):
                 pload[cpu] = int(pload_match.group(1))
                 log.debug('>>> cpu%d: %d', cpu, pload[cpu])
 
-        # target.cpufreq.load_governors()
+        for cpu, governor in old_governors.iteritems():
+            target.cpufreq.set_governor('cpu{}'.format(cpu), governor)
```

Comment:
@derkling I actually had to add the string prefix because I was getting an error - I thought devlib wasn't doing the conversion but as you say it actually does it. The value stored in the dict probably is a string, so the conversion is skipped devlib-side.